prow/cmd/deck/job_history.go

```diff
@@ -376,14 +377,28 @@ func getJobHistory(url *url.URL, config *config.Config, gcsClient *storage.Clien
 	tmpl.ResultsShown = len(shownIDs)
 	tmpl.ResultsTotal = len(buildIDs)
 
+	// concurrently fetch data for all of the builds to be shown
 	bch := make(chan buildData)
 	for i, buildID := range shownIDs {
 		go func(i int, buildID int64) {
-			bd, err := getBuildData(bkt, root, buildID, i)
+			id := strconv.FormatInt(buildID, 10)
+			dir, err := getPath(bkt, root, id, "")
+			if err != nil {
+				logrus.Errorf("failed to get path: %v", err)
+				bch <- buildData{}
+				return
+			}
+			b, err := getBuildData(bkt, dir)
```

Comment:
Let's not use single-char variable names

Reply:
And maybe we can afford some vowels?

Reply:
:)

Reply:
I suppose I can buy some vowels